ext/page_log/palm/palm.c

```diff
@@ -279,6 +341,34 @@ palm_configure_int(PALM *palm, WT_CONFIG_PARSER *env_parser, WT_CONFIG_ARG *conf
     return (ret);
 }
 
+/*
+ * palm_configure_str
+ *     Look for a particular configuration key, and return value item.
+ */
+static int
+palm_configure_str(PALM *palm, WT_CONFIG_PARSER *env_parser, WT_CONFIG_ARG *config, const char *key,
+  WT_CONFIG_ITEM *v)
+{
+    int ret;
+
+    ret = 0;
+    memset(v, 0, sizeof(*v));
+    /*
+     * Environment configuration overrides configuration used with loading the library, so check
+     * that first.
+     */
+    if ((ret = env_parser->get(env_parser, key, v)) == 0 ||
+      (ret = palm->wt_api->config_get(palm->wt_api, NULL, config, key, v)) == 0) {
+        if (v->len == 0 || v->type != WT_CONFIG_ITEM_ID)
```

Comment:
I think the config item type should be `WT_CONFIG_ITEM_STRING`. The type `WT_CONFIG_ITEM_ID` is reserved for unquoted identifiers. Identifiers can consist of very limited set of characters.

Note: When using `WT_CONFIG_ITEM_STRING`, you will need to fix `TESTUTIL_ENV_CONFIG_DISAGG_EXT` macro. Otherwise it will produce nested quotes for config string.

Reply:
Hmm, I recal; facing some issues with `WT_CONFIG_ITEM_STRING` but I will try again and see if this works